dashboard/dashboard-resource-manager.md

```diff
@@ -59,6 +59,8 @@ Before resource planning, you need to know the overall capacity of the cluster.
 
     - When the workload within the time window is too low, the following error is displayed `ERROR 1105 (HY000): The workload in selected time window is too low, with which TiDB is unable to reach a capacity estimation; please select another time window with higher workload, or calibrate resource by hardware instead` is displayed.
 
+    - The capacity estimation based on actual workload needs to Prometheus metric data, including `resource_manager_resource_unit`, `process_cpu_usage`, `tikv_cpu_quota`, and `tidb_server_maxprocs`. If the corresponding metrics data is empty, an error is displayed indicating the name of the metrics item, for example `Error 1105 (HY000): metrics 'resource_manager_resource_unit' is empty`. And `resource_manager_resource_unit` is empty when no workload, so this error will also be reported. In addition, TiKV does not monitor CPU usage  metrics on macOS, it does not support capacity estimation based on actual workload and the following error is displayed  `ERROR 1105 (HY000): metrics 'process_cpu_usage' is empty`.
```

Comment:
```suggestion
    - The capacity estimation function requires monitoring metrics data according to the actual load, including `resource_manager_resource_unit`, `process_cpu_usage`, `tikv_cpu_quota`, and `tidb_server_maxprocs`. If the corresponding monitor data is empty, an error is reported with the corresponding monitor item name, such as `Error 1105 (HY000): metrics 'resource_manager_resource_unit' is empty`. This error is also reported when `resource_manager_resource_unit` is empty when there is no load at all. In addition, because TiKV does not monitor CPU usage on macOS, it does not support capacity estimation based on actual load and will report `ERROR 1105 (HY000): metrics 'process_cpu_usage' is empty`.
```